pkg/ui/bindata.go

```diff
@@ -98,15 +98,15 @@ import (
 func bindataRead(data []byte, name string) ([]byte, error) {
 	gz, err := gzip.NewReader(bytes.NewBuffer(data))
 	if err != nil {
-		return nil, fmt.Errorf("Read %q: %v", name, err)
+		return nil, fmt.Errorf("read %q: %v", name, err)
```

Comment:
This file is automatically generated using go-bindata, so changing this manually will be overwritten next time the file is generated and will cause our CI to fail :/. Let's leave this out but keep the change from pkg/logging/http.go

Reply:
Makes sense 👍🏼  Done !